scripts/sync_codegen.py

```diff
@@ -21,6 +21,8 @@ def exec_sync(command):
     "deprecated_models.py",
     "__pycache__",
     "*.md",
+    "**/*.json",
```

Comment:
cc @csreesan – ignoring some of the files you added to substratecore (it's fine to place them where you did, i think)